### DISCUSSION TITLE
How to reduce the  size of finetuned model?

### DISCUSSION THREAD

**Discussion by cahya-wirawan** [facebookresearch/ParlAI#4371] [Q&A]
Hi, I finetuned a medium size GPT2 model (which is around 1.35GB), but the finetuned model is around 4 times bigger (5GB), is it possible to reduce it to only the necessary parameters? Thanks

**Comment by klshuster** [comment]
you can use `parlai vacuum --model-file /path/to/model` to "vacuum" the model (i.e., remove the saved optimizer states)

**Reply by cahya-wirawan** [reply]
Great, thanks, I will try it
